build_iso/sles12-sp1.obs_packages

```diff
@@ -0,0 +1,51 @@
+libstorage-ruby
```

Comment:
"Normally" a new SP has the same packages as the previous version (sles12).
This is a rule which has been defined by the SLE12 Product Managers. 
 So would it be possible to make a link in git for this file only ?